### PR TITLE
Watch MySQL and Pods on CI

### PR DESCRIPTION
**Pull request by masa213f**
I want to use CI to check:
- the status of MySQL instances and pods
- system variables of MySQL instances

Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

**Comment by yokaze** [comment]
To my opinion system variables should be checked in test cases. How do you think?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -38,10 +38,30 @@ jobs:
           sudo swapoff -a
           cat swapon.txt | tail -n+2 | awk '$2=="file" {print $1}' | sudo xargs --no-run-if-empty rm
       - run: sudo mkdir /mnt/local-path-provisioner
-      - id: run_e2e_test
-        run: cd e2e && make KIND_CLUSTER_CONFIG_SUFFIX=-actions test
+      - run: cd e2e && make setup
+      - name: watch
+        run: |
+          VARIABLES=innodb_validate_tablespace_paths
```
Comment (masa213f) [inline .github/workflows/ci.yaml:44]: This option is not present in v8.0.18 and 8v.0.20.

Comment (masa213f) [inline .github/workflows/ci.yaml:44]:
```suggestion
          VARIABLES=innodb_validate_tablespace_paths
```

**Comment by masa213f** [comment]
@yokaze 
That's true!

**Review by yokaze — APPROVED** [review]
LGTM

**Review by zoetrope — APPROVED** [review]
LGTM

**Pull request merged**